docs/installation.rst

```diff
@@ -93,18 +97,18 @@ Required dependencies for building and testing GeoCAT-viz
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     - Python 3.8+
-    - `numpy <https://numpy.org/doc/stable/>`_
-    - `xarray <http://xarray.pydata.org/en/stable/>`_
-    - `matplotlib <https://matplotlib.org/stable/index.html>`_
-    - `cartopy <https://scitools.org.uk/cartopy/docs/latest/>`_
+    - `numpy <https://numpy.org/doc/stable/>`__
+    - `xarray <http://xarray.pydata.org/en/stable/>`__
+    - `matplotlib <https://matplotlib.org/stable/index.html>`__
+    - `cartopy <https://scitools.org.uk/cartopy/docs/latest/>`__
 
 
 How to create a Conda environment for building GeoCAT-viz
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The GeoCAT-viz source code includes a conda environment definition file in
 the root directory that can be used to create a development environment containing
-all of the packages required to build GeoCAT-viz. The file :code:`conda_environment.yml`
+all of the packages required to build GeoCAT-viz. The file ``conda_environment.yml``
```

Comment:
pretty sure there's no longer a `conda_environment.yml`.  should this be `environment.yml`?